packages/client-graphql/src/client.ts

```diff
@@ -58,6 +59,13 @@ export const createCourierClient = (
     }
   }
 
+  // add client version and platform
+  headers = {
+    ...headers,
+    "x-courier-client-version": pkg.version,
+    "x-courier-client-platform": "web",
```

Comment:
hmmm, im not sure about the "web" platform here.

Reply:
any suggestion?